lib/ExportsStorage.js

```diff
@@ -136,8 +135,8 @@ export default class ExportsStorage {
           items.forEach(({ path: pathToFile, mtime }) => {
             mtimes[pathToFile] = mtime;
           });
-          const filtered = files.filter(({ path: pathToFile, mtime }) => (
-            mtime !== mtimes[pathToFile]
+          const filtered = files.filter(({ path: pathToFile, mtime, exportsAll }) => (
+            mtime !== mtimes[pathToFile] || exportsAll
```

Comment:
Revert the changes made to these two lines. The `exportsAll` option is not in use after you cleaned out that logic.

Reply:
done